### ISSUE TITLE
Improve coverage report

### ISSUE THREAD

**Issue by PicoleDeLimao** [SAP/InfraBox#79]
I'm using Infrabox to show the coverage report of my unit tests and it's great, but it'd be even greater if the there was an improvement in the UI. Currently it's a little difficult to do any analysis. My suggestions:
- Allow to sort classes by either File, Branches, Functions or Lines by clicking on each one of those categories;
- Do not list all classes. Instead, organize them by directory, and allow the user to navigate through packages. Show average statistics for each directory.
Thank you and keep up the great work!

**Comment by ib-steffen** [comment]
Yes valid points. We will work on improving the UI in the coming weeks.
